packages/e2e-tests/specs/block-hierarchy-navigation.test.js

```diff
@@ -96,8 +98,11 @@ describe( 'Navigating the block hierarchy', () => {
 		await openBlockNavigator();
 		await page.keyboard.press( 'Space' );
 
-		// Replace its content.
-		await pressKeyWithModifier( 'primary', 'A' );
```

Comment:
Can we fix the `pressKeyWithModifier` utility to work in all contexts? I know puppeteer has a global variable to check the "platform"

Reply:
Yes, possibly but does that feel like a separate PR...? I'll take a look, but I'm not over familiar with Pupeteer at this time.

Reply:
@youknowriad I can't work on this locally again because I get errors. I'm not sure how to proceed. Any advice appreciated.

Reply:
@youknowriad I invesitgated further. It appears this _isn't_ a bug with our code. Rather Puppeteer does not support modifier keys on MacOS

> NOTE On MacOS, keyboard shortcuts like ⌘ A -> Select All do not work. See #1313

https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#class-keyboard

With that in mind, should I revert this change and accept that e2e tests won't work on Mac? As long as they pass on CI then this PR should be good right?

Reply:
I think it's fine to keep this change but yeah for other tests relying on this, maybe it's fine for now, the important thing is that they pass on travis.